packages/pluggableWidgets/radio-buttons-native/src/ui/Styles.ts

```diff
@@ -0,0 +1,41 @@
+import { Style } from "@mendix/piw-native-utils-internal";
+import { TextStyle, ViewStyle } from "react-native";
+
+export interface RadioButtonsStyle extends Style {
+    containerHorizontal: ViewStyle;
+    labelTextStyle: TextStyle;
+    radioButtonItemContainerStyle: ViewStyle;
+    radioButtonItemContainerDisabledStyle: ViewStyle;
```

Comment:
Is this documented/discoverable via Atlas theme + themesource? Perhaps you don't want to add a CSS style prop-value in Atlas for the disabled state, however it is still a nicety to comment that end-users can change the disabled styling using the property `radioButtonItemContainerDisabledStyle`, over and above the native reference styling guide (docs.mendix.com).

Reply:
done